C/[01]-pengenalan-C.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Berkenalan Dengan Bahasa Pemrograman C
+created: 05-10-2021
+update: -
+---
+# 1. Berkenalan dengan C
+Bahasa Pemrograman C adalah sebuah bahasa pemrograman komputer yang bisa digunakan untuk membuat berbagai aplikasi (general-purpose programming language), 
+mulai dari sistem operasi (seperti Windows atau Linux), antivirus, software pengolah gambar (image processing), hingga compiler untuk bahasa pemrograman, 
+dimana C banyak digunakan untuk membuat bahasa pemrograman lain yang salah satunya adalah PHP.
+
+## Sejarah C
+Bahasa pemrograman C dibuat pertama kali oleh Dennis M. Ritchie pada tahun 1972. Saat itu Ritchie bekerja di Bell Labs, sebuah pusat penelitian yang berlokasi di Murray Hill, 
+New Jersey, Amerika Serikat. Ritchie membuat bahasa pemrograman C untuk mengembangkan sistem operasi UNIX. Sebelumnya, sistem operasi UNIX dibuat menggunakan 
+bahasa assembly (assembly language). Akan tetapi bahasa assembly sendiri sangat rumit dan susah untuk dikembangkan.
+
+Dengan tujuan mengganti bahasa assembly, peneliti di Bell Labs membuat bahasa pemrograman B. Namun bahasa pemrograman B juga memiliki beberapa kekurangan, 
+yang akhirnya di lengkapi oleh bahasa pemrograman C. Dengan bahasa C inilah sistem operasi UNIX ditulis ulang. Pada gilirannya, UNIX menjadi dasar dari banyak 
+sistem operasi modern saat ini, termasuk Linux, Mac OS (iOS), hingga sistem operasi Android.
+
+## God’s programming language
+Bahasa C juga dijuluki sebagai “God’s programming language” Karena banyak menginspirasi bahasa-bahasa pemrograman yang lain seperti Java, Javascript, C++, C#, PHP, dsb.
+Bahasa C memang bagus untuk memperkenalkan konsep pemrograman bagi pemula yang belum pernah coding, Usia bahasa pemrograman C memang cukup tua, 
+namun masih digunakan hingga saat ini.
+
+
+
+## Seperti apa kode C ?
+- mengeluarkan output
+  ```C
+  #include <stdio.h>
+    int main(void) {
+       printf("Hello, World!\n");
+       return 0;
+    }
+  ```
+- deklarasi variabel
+  ```C
+  char name[] = "bryan"; //tipe data yang huruf
```

Comment:
lebih pas kalo pake istilah `karakter` aja deh, `char` stands for `character` which translates to `karakter`